src/MudBlazor/Components/Radio/MudRadio.razor.cs

```diff
@@ -26,6 +26,7 @@ public partial class MudRadio<T> : MudComponentBase, IDisposable
             .AddClass($"mud-radio-dense", Dense)
             .AddClass($"mud-disabled", Disabled)
             .AddClass($"mud-checked", Checked)
+            .AddClass("mud-radio-error", MudRadioGroup?.HasErrors)
```

Comment:
```suggestion
            .AddClass("mud-error-text", MudRadioGroup?.HasErrors)
```
You can use `mud-error-text` here. Then we don't need new css